poly-commit/src/linear_codes/mod.rs

```diff
@@ -159,9 +161,9 @@ where
     S: CryptographicSponge,
     C: Config + 'static,
     Vec<F>: Borrow<<H as CRHScheme>::Input>,
-    H::Output: Into<C::Leaf>,
+    H::Output: Into<C::Leaf> + Send,
```

Comment:
btw, this still has `Send`, is that expected? I'm not sure if you said in the end that it's gonna go or not

Reply:
Yes, because of this part:
https://github.com/HungryCatsStudio/poly-commit/blob/80f44ca7df85e71d1b728127760cf1e7e847036a/poly-commit/src/linear_codes/mod.rs#L253-L259


I was talking about `Sync` not `Send`. `Sync` went away.